README.md

```diff
@@ -1,15 +1,24 @@
-# YaST - rewrite of the storage library
+# YaST - rewrite of the storage module
 
 [![Build Status](https://travis-ci.org/yast/yast-storage-ng.svg?branch=master)](https://travis-ci.org/yast/yast-storage-ng)
 [![Coverage Status](https://img.shields.io/coveralls/yast/yast-storage-ng/master.svg)](https://coveralls.io/github/yast/yast-storage-ng?branch=master)
 [![Code
 Climate](https://codeclimate.com/github/yast/yast-storage-ng/badges/gpa.svg)](https://codeclimate.com/github/yast/yast-storage-ng)
 
-yast2-storage-ng is a reimplementation of the YaST storage module (yast2-storage)
-based on the also reimplemented library for storage manager (libstorage-ng).
+yast2-storage-ng is a reimplementation of the YaST storage module
+(yast2-storage) based on the also reimplemented library for storage manager
```

Comment:
NP: we could use bold text maybe for **yast2-storage-ng** and for **yast2-storage**, in the last one without parenthesis.